msnoise/scripts/msnoise.py

```diff
@@ -459,33 +464,24 @@ def populate(fromda):
                               ' default workflow step.')
 @click.option('-r', '--recursively', is_flag=True,
               help='When scanning a path, walk subfolders automatically ?')
+@click.option('--crondays', type=int,
```

Comment:
Would it make sense to allow for a float here? For example when running the cron every 12h, you'd pass 0.5 ? (then should be allowed in the config too?)

Reply:
After your comments, I read in the release notes for 1.5 that `crondays` was already supposed to accept floats, so I'm even breaking backward compatibility here :/  I did not realise this, sorry.

But if you want more fine-grained granularity, I would advise to use a format easier to work with, so I propose (on top of allowing a float) to allow a string in the format 'Xw Xd Xh' to designated a number of weeks (i.e. 7 days, for convenience and readability), days and hours. Each value is optional here, but must be proposed in this order. The separating characters can be any blank character, and are optional.

I'm sorry, I have to merge this PR with theses changes because I won't have any time to work on it for the days to come. I hope you're OK with this proposed format.